src/components/DataSummaryCardGroup.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const DataSummaryCardGroupWrapper = styled.div`
+  display: flex;
+  &.connected-count-card-wrapper {
+    width: 100%;
+  }
+`;
+
+const SummaryCard = styled.div`
```

Comment:
we have very similar CoundCard, if you replicate code/style here, we need to change in different places when the style changed. So, please separate it using the existing CountBox in CountCard and add multiple CountBox to the group.